core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/bundle/BundleRestService.java

```diff
@@ -96,8 +96,10 @@ public BundleRestService() {
 	@GetMapping(value = "/{bundleId}", produces = { AbstractRestService.JSON_MEDIA_TYPE })
 	public Promise<Bundle> get(
 			@ApiParam(value="The bundle identifier")
-			@PathVariable(value="bundleId") final String bundleId) {
+			@PathVariable(value="bundleId") final String bundleId,
+			@ApiParam(value="expand", required = false) String expand) {
```

Comment:
`RequestParam` annotation is missing, please add it. Also, format the input parameter list to look like this:

```
    @ApiParam(value = "The bundle identifier")
    @PathVariable(value = "bundleId", required = true) 
    final String bundleId,

    @ApiParam(value = "expand", required = false)
    @PathVariable(value = "expand") 
    final String expand
```

Reply:
Done